### PR TITLE
Add image field and image tag into trade marks app model

### PR DESCRIPTION
**Pull request by smaciejs**

### CODE REVIEW

File: src/ralph/trade_marks/models.py [hunk 1]
```diff
@@ -106,6 +119,21 @@ class TradeMark(Regionalizable, AdminAbsoluteUrlMixin, BaseObject):
...
+            self.registrant_class, self.valid_to
+        )
+
+    def image_tag(self):
+        if not self.image:
+            return ""
+        return mark_safe(
+            '<img src="%s" width="150" />' % self.image.url
+        )
+    image_tag.short_description = 'Image'
```
Comment (ar4s) [inline src/ralph/trade_marks/models.py]: Please add support for translations.

File: src/ralph/trade_marks/admin.py [hunk 2]
```diff
@@ -22,6 +22,7 @@ class TradeMarksLinkedView(RalphDetailViewAdmin):
 
     class TradeMarksLinkedInline(RalphTabularInline):
         model = TradeMarksLinkedDomains
+        verbose_name_plural = "TradeMarksLinkedDomains"
```
Comment (ar4s) [inline src/ralph/trade_marks/admin.py]: 1. ``verbose_name*`` is for humans 
2. add support for translation

**Review by ar4s — CHANGES_REQUESTED** [review]

File: src/ralph/trade_marks/admin.py (hunk 2 above)
Comment (mkurek) [reply src/ralph/trade_marks/admin.py]: is it really necessary in inline (why not on model)?

**Comment by mkurek** [comment]
why can't you use existing attachments?

File: src/ralph/trade_marks/admin.py (hunk 2 above)
Comment (smaciejs) [reply src/ralph/trade_marks/admin.py]: It shouldn't be here. I'll delete this line

File: src/ralph/trade_marks/models.py (hunk 1 above)
Comment (smaciejs) [reply src/ralph/trade_marks/models.py]: Done

File: requirements/base.txt [hunk 3]
```diff
@@ -22,7 +22,7 @@ netaddr==0.7.18
 python-dateutil==2.4.2
 pytz==2015.4
 redis==2.10.3
-requests==2.20.0
+requests==2.9.1
```
Comment (szymi-) [inline requirements/base.txt]: You are downgrading requests to an insecure version - rebase your codebase to current version

File: src/ralph/trade_marks/models.py [hunk 4]
```diff
@@ -106,6 +119,21 @@ class TradeMark(Regionalizable, AdminAbsoluteUrlMixin, BaseObject):
         through='TradeMarksLinkedDomains',
     )
 
+    def __str__(self):
+        return '{}, {}, {} to {}.'.format(
```
Comment (szymi-) [inline src/ralph/trade_marks/models.py]: Maybe "expires" would be better?

File: requirements/base.txt (hunk 3 above)
Comment (smaciejs) [reply requirements/base.txt]: Sure, I'll fix it

File: src/ralph/trade_marks/models.py (hunk 4 above)
Comment (smaciejs) [reply src/ralph/trade_marks/models.py]: Why not. I'll replace this expression

**Review by ar4s — APPROVED** [review]

**Review by szymi- — APPROVED** [review]

**Pull request merged**